### PR TITLE
Add Networkswitch to Dashboard and Settings

### PR DESCRIPTION
**Pull request by FredrikOseberg**
Adding NetworkSwitch back to Dashboard and Settings. Added it back for ease of development to Dashboard. We could consider having it in the top left corner of the dashboard permanently? Though if it is going to be permanent we should change the design to fit more in line with what is currently in the Dashboard,

### CODE REVIEW

**Comment by comountainclimber** [comment]
Lets stick with the design mock on this one @FredrikOseberg 

image

Please do not feel like you need to implement any of the design above but I like the network dropdown being only on the settings page. Let me know your thoughts

**Comment by FredrikOseberg** [comment]
@comountainclimber Alright. I added the NetworkSwitch to the Dashboard as a temporary measure because it would be tedious when developing to follow this flow: 

Login -> Go To Settings -> Switch to feature being developed (IE: TokenSale) 

Compared to:

Login -> Switch on Dashboard -> Go to feature being developed.

What do you think of keeping it there for development?

Regarding whether or not to keep the NetworkSwitch only in settings permanently here are my thoughts: Personally, I the number one piece of information I'm looking for when I'm using wallets is what network I'm operating on. I would be terrified of doing any action without knowing exactly the context of where I'm operating. I'm concerned only having information about which network you are operating on obfuscated in settings could hamper usability? Perhaps we could have the switch in settings only, but display the information elsewhere in the application?

**Comment by FredrikOseberg** [comment]
Yes, that's perfect.

File: app/containers/App/Sidebar/NetworkSwitch/NetworkSwitch.jsx [hunk 1]
```diff
@@ -16,9 +18,9 @@ export default class NetworkSwitch extends Component<Props> {
   }
 
   render() {
-    const { networkId, networks } = this.props
+    const { networkId, networks, className } = this.props
     return (
-      <div id="network" className={styles.networkSwitch}>
+      <div id="network" className={classNames(styles.networkSwitch, className)}>
         <span className={styles.label}>Running on</span>
         <select
```
Comment (comountainclimber) [inline app/containers/App/Sidebar/NetworkSwitch/NetworkSwitch.jsx]: Please change this before we merge thanks

**Comment by comountainclimber** [comment]
@FredrikOseberg did you see my comment? Can we address this asap so I can merge this work in?

**Comment by FredrikOseberg** [comment]
Sorry, must have missed it. I'll take care of it.

Den fre. 17. aug. 2018, 05:14 skrev Maxwell Lasky <notifications@github.com

**Comment by FredrikOseberg** [comment]
@comountainclimber It's done.

**Comment by comountainclimber** [comment]
Nice job @FredrikOseberg ! Thank you for thia

**Pull request merged**